meetings/2021-04-26.md

```diff
@@ -0,0 +1,70 @@
+    * [Syntax for Realm?](https://github.com/solid/authentication-panel/issues/166)
+
+* [PR on HTTPSig](https://github.com/solid/authentication-panel/pull/168)
+    * Question: [HttpSig, Solid, or Signature?](https://github.com/solid/authentication-panel/issues/155)
+* [consider support for OIDC self-issuer #11](https://github.com/solid/authentication-panel/issues/163)
+
+## Present
+
+* @elf-pavlik 
+* Justin Bingham 
+* Henry Story
+* Aaron Coburn
+
+## Minutes
+
+### Discussing JSON-LD context PR
+
+**Elf**: not so clear of EricP's point. Perhaps he can make a new PR for that improvement?
+
+**Aaron**: will add that note to PR
+   Aaron quickly explains for Justin the reasons 
+   for the PR, regarding having JSON literal inside 
+   of the RDF.
+
+**Henry**: the problem was that there was "linked data" 
+   content in the JSON literal embedded in RDF, which is a bit weird. It is better if we can have a format that looks like JSON but also treats linked data in an RDF-compatible way. Hence the idea of moving to JSON-LD.
+   In the case of HttpSig, which now also has JSON embedded in the data, this is a literal JWK key. Building a whole ontology for crypto is a lot of work, and since it is a literal, I think it is ok to leave it that way in the RDF.
+   
+*Discussion on the vocabulary relating to consent screens.*
+
+### [Multiple WWW-Authenticate headers](https://github.com/solid/authentication-panel/issues/163)
+
+**Henry**: if a resource server wants to allow both HTTPSig and Solid-OIDC it may be tricky to combine it in a single WWW-Authenticate header.
+**Aaron**: I don't see problems with multiple WWW-Authenticate headers. Problems may come if we try to use multiple Authorization headers. The questions that arise are: should the server verify all of the tokens or only some? 
+**Henry**: I don't think this issue is crucial right now, but since we do have HttpSig and OIDC as well as Basic, it will pop up. So I thought it would be good to gather the things to think about someplace.
+
+### [Sending Access control rules in the body of 401?](https://github.com/solid/authentication-panel/issues/167)
+
+**Pavlik**: There is a problem here when accessing resources directly in the browser, as these need to be displayed in HTML. I want to make primer PR relating to accessing resources directly with a browser. It would put some requirements on what 401 response should include (edit: at least for `text/html`)
+**Aaron**: I would be careful about privacy with returning that information.
+  Back in the days, during login, sites would respond with 'bad password' or 'unknown username'. Now they just reject and don't leak any information related to user accounts.
+**Henry**: yes, I am not too keen on putting ACL data in the body of a 401 for the reasons given above. I just thought it would be worth gathering those thoughts, as it is an option to consider.
```

Comment:
https://github.com/solid/authentication-panel/issues/167#issuecomment-831500188

Reply:
I updated the text in reply to @acoburn above regarding privacy. See commit #a888d1aa079e120959e12af95437691be31d6d19